TicketManagement/src/main/java/com/endava/ticketmanagement/repository/EventRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.endava.TicketManagement.repository;
+
+import com.endava.TicketManagement.repository.model.Event;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface EventRepository extends JpaRepository<Event,Long> {
+    public Event findByEventName(String name);
```

Comment:
Like it, dar te rog sa te gandesti sa mai adaugi poate si alte metode care te-ar ajuta la endpoints